master/internal/api_user.go

```diff
@@ -170,12 +173,19 @@ func (a *apiServer) GetUsers(
 		nameFilterExpr := "%" + req.Name + "%"
 		query.Where("u.display_name ILIKE ? OR u.username ILIKE ?", nameFilterExpr, nameFilterExpr)
 	}
-	if req.Admin != nil {
+	if req.Admin != nil && !config.GetAuthZConfig().IsRBACEnabled() {
```

Comment:
do we need this check? It feels weird to me to prevent searcher on admin just since rbac is enabled

Can the front end not filter by admin when rbac is enabled?

Reply:
Of course